x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/constants.ts

```diff
@@ -101,6 +101,13 @@ export const AVAILABLE_SOURCE_CUSTOM_SOURCE_BUTTON = i18n.translate(
   }
 );
 
+export const CONFIG_COMPLETED_PRIVATE_SOURCES_DISABLED_LINK = i18n.translate(
+  'xpack.enterpriseSearch.workplaceSearch.contentSource.configCompleted.privateDisabled.link',
```

Comment:
As per the i18n guidelines, can we change this to:
```suggestion
  'xpack.enterpriseSearch.workplaceSearch.contentSource.configCompleted.privateDisabledLinkLabel',
```

Reply:
RE: "id format" replied here: https://github.com/elastic/kibana/pull/112136#issuecomment-919985697